ramls/hridsettings.json

```diff
@@ -24,6 +24,11 @@
       "description": "The HRID settings for items",
       "type": "object",
       "$ref": "hridsetting.json"
+    },
+    "commonRetainLeadingZeroes" : {
+      "type": "boolean",
+      "description": "If true, leading zeros should be applied for HRIDs",
+      "default": true
     }
```

Comment:
Does this setting apply to all HRIDs irrespective of the record type?

Reply:
Yes.  One parameter will be used for Instances/Holdings and Items (one checkbox on UI).

Reply:
Then please remove the per record-type parameters that were added.

Reply:
Ready. Per record-type parameters were deleted.

Reply:
@Aliaksandr-Fedasiuk Thank you